### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by alexanderhejazi**

### CODE REVIEW

File: config/env.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+// Google Captcha
```
Comment (alexanderhejazi) [inline config/env.php:3]: Are we even using a captcha somewhere? Or is this taken from the BB code? If we're not using it then we can just delete it all together 😄 @lanceboyer1

File: config/env.php [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+$_ENV['GOOGLE_CAPTCHA_SITE_KEY'] = "6LdNijMdAAAAANctKNaUespZ-uPdjxFAVXUr_VYo";
+$_ENV['GOOGLE_CAPTCHA_SECRET_KEY'] = "6LdNijMdAAAAAJ-h6RJEJSK3UGcQNzF9gzxMpepC";
+$_ENV['GOOGLE_CAPTCHA_ENDPOINT'] = "https://www.google.com/recaptcha/api/siteverify?secret={$_ENV['GOOGLE_CAPTCHA_SECRET_KEY']}" . "&response=";
+
+// Database
+$_ENV['DB_HOST'] = "localhost";
+$_ENV['DB_USER'] = "root";
+$_ENV['DB_PASS'] = "";
```
Comment (alexanderhejazi) [inline config/env.php:11]: Please - never store real credentials in a public repository. There are bots scanning repositories to look for passwords. @lanceboyer1

File: core/functions.php [hunk 3]
```diff
@@ -0,0 +1,24 @@
+<?php
```
Comment (alexanderhejazi) [inline core/functions.php:1]: If we do not use the Google captcha then this file is not needed at all. Please let me know if you need captcha for now, or not 😄

File: process-form.php [hunk 4]
```diff
@@ -1,7 +1,7 @@
 <?php
 $servername = "localhost";
 $username = "root";
-$password = "W1ll1@m!!";
+$password = "";
```
Comment (alexanderhejazi) [inline process-form.php:4]: Again - please no credentials in a public Git repo 😄

**Review by alexanderhejazi — COMMENTED** [review]
Please check my comments @lanceboyer1

File: core/functions.php (hunk 3 above)
Comment (alexanderhejazi) [reply core/functions.php:1]: I've removed the captcha. So if you merge this it won't be in your code anymore.

File: config/env.php (hunk 1 above)
Comment (alexanderhejazi) [reply config/env.php]: I've removed the captcha. So if you merge this it won't be in your code anymore.

**Review by lanceboyer1 — APPROVED** [review]

**Pull request merged**